tests/test_events.py

```diff
@@ -51,58 +54,28 @@ def test_agenda_pdf_form_error():
         assert agenda_pdf_form.is_valid() == False
 
 
-def test_updates_made_true(event, event_document):
```

Comment:
We can't keep this exact behavior and instead will do what we describe her: https://github.com/datamade/la-metro-councilmatic/issues/509